core/tests/performance_framework/perf_services.py

```diff
@@ -56,15 +66,46 @@ class SeleniumPerformanceDataFetcher(object):
 
     # Duration (in seconds) to wait for the complete page to load, otherwise
     # XHR requests made post initial page load will not be recorded.
-    WAIT_DURATION_SECS = 3
+    DEFAULT_WAIT_DURATION_SECS = 3
+
+    BASE_URL = 'http://localhost:%d' % feconf.PERFORMANCE_TESTS_SERVER_PORT
```

Comment:
I don't think the PERFORMANCE_TESTS_SERVER_PORT constant has to be in feconf; who else uses it? Would it be better to have it as a top-level constant in this file?